.travis.yml

```diff
@@ -21,7 +21,7 @@ matrix:
     - php: '7.2'
```

Comment:
As far as I remember, this part of the matrix should contains all the supported PHP versions minus the latest which has code coverage enabled.